examples/feeds/get_feed_items_of_feed_item_set.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example gets all feed items of the specified feed item set.
+
+To create a new feed item set, run create_feed_item_set.py.
+To link a feed item to a feed item set, run link_feed_item_set.py.
+"""
+
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+from google.ads.google_ads.util import ResourceName
+
+
+def main(client, customer_id, feed_id, feed_item_set_id):
+    ga_service = client.get_service("GoogleAdsService", version="v6")
+    feed_item_set_service = client.get_service("FeedItemSetService",
+                                               version="v6")
+
+    query = ("SELECT feed_item_set_link.feed_item FROM feed_item_set_link "
+             "WHERE feed_item_set_link.feed_item_set = '%s'"
+             % feed_item_set_service.feed_item_set_path(
```

Comment:
Use f-strings instead of the `%` operator for interpolation. And it might be easier to set this to its own variable than to make the call to format_composite inside the f-string brackets.

Reply:
Yes, I avoided using f-strings at first because of the function calling.
I'll create a new variable instead.